### ISSUE TITLE
Fix Jetpack composable preview renderer

### ISSUE THREAD

**Issue by Ehsan-Tal** [Ehsan-Tal/Habitua#33]
Why is this an issue ? Because the renderer cannot deliver the view model or interact with the view model factory

- [x] Fix the viewmodel issue stopping us from rendering previews (Separate the routes and screens ?)
- [x] Fix preview issue in Habit Edit
- [x] Fix preview in Home

**Comment by Ehsan-Tal** [comment]
I found the cause of the issue ! Remember to mind the stack and interrogate it step by step.

PROBLEM: Previews weren't rendering because it couldn't instantiate a view model

RAMBLE:
I decoupled the view model from the entry screen - something I should exercise as I build - to no avail.

Turns out, I had coupled `Habitua Theme` with the settings view model to change the colour schemes to dark mode light mode. 

SOLUTION:
Added a composable `Theme` that did not call the view models. It's called `PreviewHabituaTheme` and I have also added multiple previews to account for both the dark and light mode - and I have named and grouped them.

Bug solved ! Now get to the other pages.
